src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1084,9 +1089,8 @@ FixedwingPositionControl::control_auto_position(const float control_interval, co
 
 	float tecs_fw_thr_min;
 	float tecs_fw_thr_max;
-	float tecs_fw_mission_throttle;
 
-	float mission_throttle = _param_fw_thr_cruise.get();
+	float mission_throttle = _param_fw_thr_trim.get();
 
 	if (PX4_ISFINITE(pos_sp_curr.cruising_throttle) &&
```

Comment:
maybe a discussion for another thread.. but wasn't there some discussion about removing the ability to set cruising throttle through the position setpoint?

Here this doesnt even make it to TECS now that the trim_throttle is not passed through the update function, so there will be a speed weight of 2, but throttle will be at trim? Why/when is this desired?

Reply:
Yes I thought we aligned on not using the throttle setpoint but always control airspeed through the airspeed setpoint, also in airspeed-less mode. But then how do we support gliding..maybe pass a boolean flag or it (e.g. `gliding_on`).

Reply:
>But then how do we support gliding

What do you mean by supporting gliding? Like a specific mode for shutting off the motor and relying on updrafts? Or just allowing the throttle to be decreased to zero if TECS thinks that's an ok state to achieve the given flight path setpoint?

In the latter.. I would say just put min throttle to zero, right? For the prior .. this seems to be something we should think about more carefully than just letting users say "glide" and then it shuts off the motor and starts dropping out of the sky (though along some waypoint line). Unless I am missing something here?